src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -52,9 +52,19 @@ namespace AppInstaller::CLI::Execution
         LogPath,
         InstallerArgs,
         CompletionData,
+        InstalledPackageVersion,
         Max
     };
 
+    // bit masks used as Context flags
+    enum class ContextFlag : int
+    {
+        None = 0x0,
+        InstallerExecutionUseUpdate,

```

Comment:
>InstallerExecutionUseUpdate [](start = 8, length = 27)

As a flags enum, we will have to specify the values after this one, so we should still have it be 0x1.

Reply:
I guess I forgot to add back when modifying..

---
In reply to: [504133540](https://github.com/microsoft/winget-cli/pull/588#discussion_r504133540) [](ancestors = 504133540)